Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2017 Intel Corporation
```

Comment:
I think +1 for that - there are dockerfiles we should add for the metrics tests for instance, and having unique names would be good (sure, that means we would then need a `docker build -f xxx` than defaulting to the dockerfile default name - but it is either that or we have a separate directory for each dockerfile.